css/properties/animation-timing-function.json

```diff
@@ -170,6 +170,57 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "jump": {
+          "__compat": {
+            "description": "<code>jump-</code> keywords",
```

Comment:
Are the `jump-*`s are keywords of `animation-timing-function` directly? For example, is `animation-timing-function: jump-start` valid and meaningful? Looking at the spec, it looks to me that it's only a step position of `step()`.

If I've understood correctly, then I'd expect this feature and description to make that relationship clearer (for example, naming the feature `jump_step_positions` and describing it as `<code>jump-</code> step positions for <code>step()</code>` or something like that)